examples/experimental/dagster-airlift/dagster_airlift/core/airflow_instance.py

```diff
@@ -357,6 +358,7 @@ def file_token(self) -> str:
         return self.metadata["file_token"]
 
 
+@whitelist_for_serdes
```

Comment:
this does concern me ever so slightly, means we might need to start constraining our field retrieval

Reply:
also, this should move into the serialized classes file